js/chat/chat.ts

```diff
@@ -369,6 +372,7 @@ class ChatContainer extends LightElement {
 
     if (message.chunk_type === "message_end") {
       lastMessage.removeAttribute("is_streaming");
+      lastMessage.setAttribute("content", message.content);
```

Comment:
After chatting with @wch, we discovered/remembered this is needed for @wch's hand-rolled throttling implementation, but it's also useful if want to transform the content when the message is `done`, for example:

```python
from shiny.express import ui

chat = ui.Chat(id="chat")
chat.ui()


@chat.transform_assistant_response
def transform(content: str, chunk: str, done: bool) -> str | None:
    if done:
        return content + "...DONE!"
    else:
        return content


@chat.on_user_submit
async def _():
    await chat.append_message_stream(("Basic ", "response"))
```